README.md

```diff
@@ -376,20 +376,13 @@ new command, add it to `_processCommand`, write and call processor for it.
 
 ### Publish new `npm` release
 
-1. On the `main` branch, bump the chromium-bidi version number in `package.json`:
+1. Open a PR bumping the chromium-bidi version number in `package.json` for review:
 
    ```sh
    npm version patch -m 'Release v%s'
```

Comment:
Do we need to add `--no-git-tag-version` here, given the other changes? (Otherwise we’d be creating a tag on some local branch.)

Reply:
Good point, added!